lib/shopify-cli/heroku.rb

```diff
@@ -82,6 +86,18 @@ def heroku_command
       local_path = File.join(ShopifyCli::CACHE_DIR, 'heroku', 'bin', 'heroku').to_s
       if File.exist?(local_path)
         local_path
+      elsif @ctx.windows?
+        # Check if Heroku exists in the Windows registry and run it from there
+        require 'win32/registry'
+        begin
+          windows_path = Win32::Registry::HKEY_CURRENT_USER.open('SOFTWARE\heroku') do |reg|
```

Comment:
WE ARE HITTING THE REGISTRY. THIS IS POD RACING!

Reply:
😱